### PR TITLE
Docs guides

### PR DESCRIPTION
**Pull request by uosjapuelks**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ Feature | Command Format |
 -----------|-----------------
 [Add an item](#add-an-item-into-fridget-add) | `add ITEM_NAME /EXPIRY_DATE`
 [Remove an item](#remove-an-item-from-fridget-remove) | `remove ITEM_NAME`
+[Update the quantity of an item]() | `update ITEM_NAME`
```
Comment (BryanElmer) [inline docs/UserGuide.md:61]: link is not working yet

File: docs/UserGuide.md [hunk 2]
```diff
@@ -194,6 +195,19 @@ You have successfully removed:
 __________________________________________
 ```
 
+<ins>If there is only one item with a name containing or with different upper or lower case to `ITEM_NAME`</ins>:
```
Comment (alvynben) [inline docs/UserGuide.md:198]: Could this be changed to "If there is only one item with a similar spelling (different capitalization / has more letters)"?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -249,6 +264,124 @@ __________________________________________
...
+__________________________________________
+How many of burger do you have left?
+__________________________________________
+USER INPUT: 32
+__________________________________________
+Quantity of burger is now 32.
+__________________________________________
+```
+
+<ins>If the intended value to be update is Zero</ins>:
```
Comment (alvynben) [inline docs/UserGuide.md:355]: Typo. Should be to "update**d** is **z**ero"

File: docs/UserGuide.md [hunk 4]
```diff
@@ -249,6 +264,124 @@ __________________________________________
...
+__________________________________________
+How many of fish burger do you have left?
+__________________________________________
+USER INPUT: 32
+__________________________________________
+Quantity of burger is now 32.
+__________________________________________
+```
+
+<ins>If there are multiple quantities of the same item</ins>:
```
Comment (alvynben) [inline docs/UserGuide.md:336]: Could this be changed to "If there are multiple items of the same name?"

**Review by alvynben — APPROVED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (uosjapuelks) [reply docs/UserGuide.md]: done

File: docs/UserGuide.md (hunk 3 above)
Comment (uosjapuelks) [reply docs/UserGuide.md]: done

File: docs/UserGuide.md (hunk 4 above)
Comment (uosjapuelks) [reply docs/UserGuide.md]: deon

File: docs/UserGuide.md (hunk 1 above)
Comment (uosjapuelks) [reply docs/UserGuide.md]: fixed

**Pull request merged**